bitrise.yml

```diff
@@ -79,6 +79,21 @@ workflows:
     before_run: []
 
 
+  configure-sentry:
+    steps:
+    - script@1:
+        title: Configure Sentry
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            set -x
+            /usr/libexec/PlistBuddy -c "Add :SentryDSN string ${SENTRY_DSN}" Blockzilla/Info.plist
+    meta:
+      bitrise.io:
+        stack: osx-xcode-13.2.x
+    before_run: []
```

Comment:
Nit: I think this is not needed

Reply:
I'll leave it in this PR and file a followup - if we don't need it here then we can probably remove it too from a bunch of other tasks.